<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeWidget</class>
 <widget class="QWidget" name="NoticeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton#toolButtonTitle{  
	background-image: url(:/icon/image/他1.png);
	background-repeat: no-repeat;
    background-origin: content;
	background-position: center;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{
	background-color: rgb(46, 54, 65);
}
QLabel{
	font: 500 14pt &quot;微软雅黑&quot;;
    color: white;
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;公告&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TagWidget" name="widgetTagBar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QToolButton" name="toolButtonMin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
                                             border-image: url(:/login/image/login/小zc.png);
                                         }
                                         QToolButton:hover
                                         {
                                             border-image: url(:/login/image/login/小move.png);
                                         }
                                         QToolButton:pressed{
                                             border-image: url(:/login/image/login/小down.png);
                                         }</string>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>23</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
                                             border-image: url(:/login/image/login/da-zc.png);
                                         }
                                         QToolButton:hover
                                         {
                                             border-image: url(:/login/image/login/da-move.png);
                                         }
                                         QToolButton:pressed{
                                             border-image: url(:/login/image/login/da-down.png);
                                         }</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton{
                                           border-image: url(:/login/image/login/关.png);
                                       }
                                       QToolButton:hover
                                       {
                                           border-image: url(:/login/image/login/关move.png);
                                       }
                                       QToolButton:pressed{
                                           border-image: url(:/login/image/login/关down.png);
                                       }</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>110</width>
            <height>12</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton,#pushButtonSendNotice{
	font: 12pt &quot;微软雅黑&quot;;
    color: #009dda;
    border-style: solid;
    border-width: 2px;
    border-color: #e4e4e4;
	background-color: #99ccff;
	min-height:30px;
	max-height:30px;
}
QPushButton#pushButton:hover,#pushButtonSendNotice:hover{
	font: 12pt &quot;微软雅黑&quot;;
    color: #009dda;
    border-style: solid;
    border-width: 2px;
    border-color: #009dda;
	background-color: #99ccff;
}
QPushButton#pushButton:checked,#pushButtonSendNotice:checked{
	font: 12pt &quot;微软雅黑&quot;;
    color: #ffffff;
    border-style: solid;
    border-width: 2px;
    border-color: #009dda;
	background-color: #009dda;
}QPushButton#pushButton:pressed,#pushButtonSendNotice:pressed{
	font: 12pt &quot;微软雅黑&quot;;
    color: #ffffff;
    border-style: solid;
    border-width: 2px;
    border-color: #009dda;
	background-color: #009dda;
}
QLabel#labelusername{
	font: 12pt &quot;微软雅黑&quot;;
    color: black;
}
QLabel#labelsign{
	font: 10pt &quot;微软雅黑&quot;;
    color: #999999;
}
QWidget#widget{
	background-color: qlineargradient(spread:reflect, x1:0.471591, y1:0, x2:0.463, y2:0.994, stop:0 rgba(242, 242, 242, 255), stop:1 rgba(255, 255, 255, 255));
}  
QPushButton#pushButtonSeeNotice,#pushButtonDraftNotice{
	font: 12pt &quot;微软雅黑&quot;;
    color: #666666;
	background-color: #ffffff;

}
QPushButton#pushButtonSeeNotice:hover,#pushButtonDraftNotice:hover{
	font: 500 12pt &quot;微软雅黑&quot;;
    color: #009dda;
	background-color: #ffffff;

}
QPushButton#pushButtonSeeNotice:pressed,#pushButtonDraftNotice:pressed{
	font: 500 12pt &quot;微软雅黑&quot;;
    color: #009dda;
border-width: 0px; 
   border-style: solid;
    border-color: #e4e4e4;
		background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(242, 242, 242, 255));
	background-image: url(:/notice/image/Notice/u397.png);
	background-position:left center;
	background-repeat: no-repeat;
	background-origin: content;
}
QPushButton#pushButtonSeeNotice:checked,#pushButtonDraftNotice:checked{
	font: 500 12pt &quot;微软雅黑&quot;;
    color: #009dda;
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(242, 242, 242, 255));
border-width: 0px;  
  border-style: solid;
    border-color: #e4e4e4;
	background-image: url(:/notice/image/Notice/u397.png);
	background-position:left center;
	background-repeat: no-repeat;
	background-origin: content;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#widget_5{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.994, stop:0 rgba(220, 240, 254, 255), stop:1 rgba(205, 234, 254, 255));
}
</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QToolButton" name="toolButtonTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="labelusername">
                 <property name="text">
                  <string>用户名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonTitlIcon">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	padding-right: 15px;
}
QPushButton::menu-indicator {
    color: black;
    subcontrol-position: right center;
    subcontrol-origin: padding;
    right: 0px;
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../image.qrc">
                   <normaloff>:/statu/image/status/在线.png</normaloff>:/statu/image/status/在线.png</iconset>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="labelsign">
               <property name="text">
                <string>个性签名</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
border: 1px solid blue;

background-image: url(:/icon/image/search.png);
background-position:right center;
background-repeat: no-repeat;
min-height: 25px;

border-width:1px;
border-radius: 5px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonSendNotice">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>116</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string>发公告</string>
            </property>
            <property name="icon">
             <iconset resource="../image.qrc">
              <normaloff>:/notice/image/Notice/mail_sender_u414.png</normaloff>:/notice/image/Notice/mail_sender_u414.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../image.qrc">
              <normaloff>:/notice/image/Notice/u422.png</normaloff>:/notice/image/Notice/u422.png</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#widget_3 {

 }
QWidget#widget_3 {
	padding:5px;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonSeeNotice">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>查公告</string>
             </property>
             <property name="icon">
              <iconset resource="../image.qrc">
               <normaloff>:/notice/image/Notice/mail_draft_u412.png</normaloff>:/notice/image/Notice/mail_draft_u412.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelSeeNum">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonDraftNotice">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>草稿箱</string>
             </property>
             <property name="icon">
              <iconset resource="../image.qrc">
               <normaloff>:/notice/image/Notice/mail_receiver_u410.png</normaloff>:/notice/image/Notice/mail_receiver_u410.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDraftNum">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>466</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="contentStackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#contentStackedWidget{
	background-color: rgb(255, 255, 255);

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagWidget</class>
   <extends>QWidget</extends>
   <header location="global">notice/tagwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
